server/pbench/bin/pbench-sync-satellite

```diff
@@ -211,43 +246,28 @@ for host in $hosts ;do
     if [ -s $logdir/$host/ok-checks.log ] ;then
         state_list=$(cat $logdir/$host/ok-checks.log | sed -n 's/: OK//p')
         for x in $state_list; do
-            echo "$host/TO-SYNC/$x" >> $LOGSDIR/change_state.$prefix.log
+            echo "$host/TO-SYNC/$x" >> $LOGSDIR/change_state.$remote_prefix.log
```

Comment:
According to the new naming convention, this line should be

`echo "$host/TO-SYNC/$x" >> ${state_change_log}` 

otherwise, it won't be saving the results in the right position. And we won't get anything while checking whether to call the change-state script or not.

Reply:
Thanks, I'll fix this in an update.